src/js/actions/layers.js

```diff
@@ -1544,19 +1544,22 @@ define(function (require, exports) {
      * @param {Document} document
      * @param {Immutable.Iterable.<Layer>} layers
      * @param {number} opacity Opacity as a percentage
-     * @param {boolean=} coalesce Whether to coalesce this operation's history state
+     * @param {object} options
+     * @param {boolean=} options.coalesce Whether to coalesce this operation's history state
      * @return {Promise}
      */
-    var setOpacity = function (document, layers, opacity, coalesce) {
+    var setOpacity = function (document, layers, opacity, options) {
         layers = layers.filterNot(function (layer) {
             return layer.isBackground;
         });
+        
+        options = _.merge({ coalesce: false }, options);
```

Comment:
Oh, maybe because you want a fresh object reference below. It still seems like the first param could be an empty object literal, but again it probably doesn't really matter.